freqtrade/misc.py

```diff
@@ -81,13 +81,13 @@ def file_load_json(file):
         gzipfile = file
     # Try gzip file first, otherwise regular json file.
     if gzipfile.is_file():
-        logger.debug('Loading ticker data from file %s', gzipfile)
-        with gzip.open(gzipfile) as tickerdata:
-            pairdata = json_load(tickerdata)
+        logger.debug(f"Loading historical data from file {gzipfile}")
```

Comment:
```suggestion
        logger.debug(f"Loading data from file {gzipfile}")
```

I'd make this (and the below) more generic ... 
This is only used for trades atm, not for ohlcv data anymore ...

Reply:
But trades (from gzipfile) are still "historical data"?

Reply:
Would leave it as it is, trades loaded from disk are also historical data.